git_machete/long_docs.py

```diff
@@ -0,0 +1,934 @@
+from typing import Dict
```

Comment:
Actually, a high-level question: since these docs are auto-generated, should they be tracked under git at all? I'd either expect:
1. that these files are git-ignored, only created once `tox -e docs` runs OR
2. that these files are tracked as under git, but then CI only checks if they're identical to what you'd get from the existing .rst docs (e.g. just checks if they're kept up to date with .rst)

Actually, I'd even say that the latter option will give us more insight/control over what's kept in these files... although it's rather unorthodox (typically what I see in various ecosystems is rather (1.))

Reply:
Alright, lets go with second option

Reply:
Yeah, we've agreed to (2.) coz it's more future-proof (what if some yet-unhandled formatting in RST ends up looking weird in docs.py?)